common/src/test/java/tachyon/security/authentication/SimpleAuthenticationProviderImplTest.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.security.authentication;
+
+import org.junit.Test;
+
+public class SimpleAuthenticationProviderImplTest {
+
+  @Test
+  public void createSimpleAuthenticationProviderTest() throws Exception {
+    new SimpleAuthenticationProviderImpl();
+  }
+
+  @Test
+  public void anyUserAllowConnectTest() throws Exception {
+    AuthenticationProvider authenticationProvider = new SimpleAuthenticationProviderImpl();
```

Comment:
since what `createSimpleAuthenticationProviderTest` does is a subset of `anyUserAllowConnectTest`, I think it is unnecessary to keep `createSimpleAuthenticationProviderTest`